### DISCUSSION TITLE
Can't login after adding with LDAP

### DISCUSSION THREAD

**Discussion by bigmaneric97** [minio/minio#14825] [Q&A] [answered]
I'm having a issue logging in with my access key after adding LDAP. Is there a way to remove the ldap settings without having to log into the console?

Thank you.

**Comment by harshavardhana** [comment, marked as answer]
> I'm having a issue logging in with my access key after adding LDAP. Is there a way to remove the ldap settings without having to log into the console?

just use `mc` to change those settings
